src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -27,6 +27,7 @@ public class MessageBoxInstancesController : ControllerBase
         /// Initializes a new instance of the <see cref="MessageBoxInstancesController"/> class
         /// </summary>
         /// <param name="instanceRepository">the instance repository handler</param>
+        /// <param name="instanceEventRepository">the instance event repository service</param>
```

Comment:
is this added because we are looking to retrieve the instance events for the message box from this controller rather than the instanceEvents controller?